tests/attr/test_integrated_gradients_basic.py

```diff
@@ -253,86 +264,101 @@ def _assert_batched_tensor_multi_input(
 
     def _compute_attribution_and_evaluate(
         self,
-        model,
-        inputs,
-        baselines=None,
-        target=None,
-        additional_forward_args=None,
-        type="vanilla",
-        approximation_method="gausslegendre",
-    ):
+        model: nn.Module,
```

Comment:
nit: Can we add "from torch.nn import Module" and just use Module here? That's the way we've done it in other files, so it would be nice to remain consistent here.

Reply:
Done.